routes/recordRoutes.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const router = express.Router()
+const recordController = require('../controllers/recordController')
+
+router.post('/', recordController.createRecord)
+router.put('/', recordController.editRecord)
+router.delete('/', recordController.deleteRecord)
+router.get('/list', recordController.listRecords)
```

Comment:
es curioso que a pesar de ser una ruta GET, pasas datos por el Body (o al menos esa es la logica de tu editRecord Controller). Eso no es buena practica, deberias jalar los endpoints de los query parameters.

Reply:
yo se que no es buena practica, pero hace como 2 sprints me pediste que los datos del file los pasara por el body, y no por el query, thats why i did it in that way, perdon